packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -95,6 +96,8 @@ export class GaussianSplattingMesh extends Mesh {
         new SubMesh(0, 0, 4, 0, 6, this);
 
         this.setEnabled(false);
+        // webGL2 and webGPU support for RG texture with float16 is fine. not webGL1
+        this._useRGBACovariants = !this.getEngine().isWebGPU && this.getEngine().version === 1.0;

```

Comment:
You don't have to support WebGL1, see @deltakosh answer offline!

Reply:
without this support, checks will not pass.